client/src/Components/Shared/StarsForm.jsx

```diff
@@ -16,13 +16,17 @@ class Stars extends React.Component {
 
   setRating(stars) {
     this.setState({
-      numberOfStarts: stars,
-      starOnHover: stars - 1
+      starSelected: stars,
+      starOnHover: stars - 1,
+      numberOfStars: stars + 1
     });
   }
 
   setHover (index) {
+<<<<<<< HEAD:client/src/Components/Shared/StarsForm.jsx
+=======
     //console.log(index)
+>>>>>>> 9ecad9a24a287274d5bee422332bb0e78eb90a65:client/src/Components/Shared/Stars.jsx
```

Comment:
Can you check if these words are actually in the file? If so can you delete them, the commented console log is fine if you want it just the HEAD, symbols, and the hash looking thing at the bottom